src/api/layers/__tests__/KMLLayer.class.spec.js

```diff
@@ -0,0 +1,104 @@
+import { describe, expect, it } from 'vitest'
+
+import { LayerAttribution } from '@/api/layers/AbstractLayer.class'
+import KMLLayer from '@/api/layers/KMLLayer.class'
+
+describe('KMLLayer', () => {
+    it('should create a KMLlayer instance with default values', () => {
+        const fileId = '-uQyFMtTSCWC_9rZE3EJ6B'
+        const kmlFileUrl = `https://sys-public.dev.bgdi.ch/api/kml/files/${fileId}`
+        const layer = new KMLLayer(kmlFileUrl)
+        expect(layer.kmlFileUrl).toBe(kmlFileUrl)
+        expect(layer.visible).toBe(true)
```

Comment:
with boolean there's a `to.be.true` and `to.be.false` if you want a shorthanded 😉

Reply:
done